src/main/webapp/view/lecture-view.css

```diff
@@ -54,6 +54,10 @@ body {
   padding: 15px;
 }
 
+#discussion .spinner-border {
+  margin-left: 45%;
```

Comment:
ditto here

Reply:
(resolved in chat) margin-auto unfortunately doesn't work :(